internal/parseutil/filereader_test.go

```diff
@@ -0,0 +1,48 @@
+package parseutil
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestLineResolution(t *testing.T) {
```

Comment:
Let's call this `TestFileReader` if we're testing all of `FileReader`, or break it up into smaller functions with more specific names if that's simpler.